### DISCUSSION TITLE
Editable Text Model

### DISCUSSION THREAD

**Discussion by amanver16** [pikasojs/pikaso#65] [Ideas]
I am using this wonderful js library in my project. Everything works great. However, I have a project requirement to edit the text along with resize. In case of label, I can edit it by double clicking, but cannot resize like text model. It only allows me to resize width.

How can I resize the label like text from all sides and corners ?
OR
How can I make text (shape) editable like label ?

**Comment by raminious** [comment]
Hi @amanver16 
It makes sense.
It might be possible that I add a flag to the core library to let you resize the text as you mentioned.
So I'm adding this issue to current release for more investigations

**Reply by raminious** [reply]
Oh, actually this is a discussion. please create an issue.

**Comment by raminious** [comment]
resolved in https://github.com/pikasojs/pikaso/issues/66
